app/instance-initializers/browser/ember-href-to.js

```diff
@@ -1,17 +1,29 @@
 import Em from 'ember';
 import HrefTo from 'ember-href-to/href-to';
 
+let hrefToClickHandler;
+function closestLink(el) {
+  if (el.closest) {
+    return el.closest('a');
+  } else {
+    el = el.parentElement;
+    while (el && el.tagName !== 'A') {
+      el = el.parentElement;
+    }
+    return el;
+  }
+}
 export default {
   name: 'ember-href-to',
-  initialize: function(applicationInstance) {
-    let $body = Em.$(document.body);
-    $body.off('click.href-to', 'a');
-    
-    $body.on('click.href-to', 'a', (e) => {
-      let hrefTo = new HrefTo(applicationInstance, e);
-      hrefTo.maybeHandle();
-
-      return true;
-    });
+  initialize(applicationInstance) {
+    document.body.removeEventListener('click', hrefToClickHandler);
```

Comment:
Actually the ordering of the instructions is important. 

In acceptance tests this initializer is called over and over. On each run we want to remove the event handler of the previous run and add the event handler of the new one.